onnxruntime/hosting/environment.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef ONNXRUNTIME_HOSTING_ENVIRONMENT_H
+#define ONNXRUNTIME_HOSTING_ENVIRONMENT_H
+
+#include <memory>
+
+#include "core/framework/environment.h"
+#include "core/common/logging/logging.h"
+#include "core/session/inference_session.h"
+#include "log_sink.h"
+
+namespace onnxruntime {
+namespace hosting {
+
+class HostingEnvironment {
+ public:
+  HostingEnvironment();
+
+  const onnxruntime::logging::Logger& GetLogger();
```

Comment:
Google style guide wants const after `GetLogger()` 

https://google.github.io/styleguide/cppguide.html#Use_of_const

Reply:
For this case, the return value is `const onnxruntime::logging::Logger&`. So I will leave it as-is. But the `GetSessin()` should be a `const` method. I will update it.